src/rust/src/backend/ec.rs

```diff
@@ -501,17 +422,226 @@ impl ECPublicKey {
     }
 }
 
+#[pyo3::prelude::pyclass(frozen, module = "cryptography.hazmat.bindings._rust.openssl.ec")]
```

Comment:
We've been generally doing this for rust objects, but should we actually claim the module is `cryptography.hazmat.primitives.asymmetric.ec.EllipticCurvePrivateNumbers` since that's the only public place we expose this class now?